releases/v0.10.0.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes for v0.10.0
+
+Release Date: September 25th, 2024
+
+This release is based on [3.9.0](https://github.com/kata-containers/kata-containers/releases/tag/3.9.0) of Kata Containers
+and [v0.10.0](https://github.com/confidential-containers/enclave-cc/releases/tag/v0.10.0) of enclave-cc.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for
+definitions of the acronyms used in this document.
+
+## What's new
+
+* Support to pull and verify `cosign`-signed images
+* Trusted image storage on guest
+* Guest support for image pull
```

Comment:
Guest image pull was already supported. Which PR are you referring to here?

Reply:
My mistake. This is not new support; this a re-work of the API from [here](https://github.com/confidential-containers/guest-components/pull/608), which we can leave off the notes.